templates/corporate/remote_realm_billing_finalize_login_confirmation.html

```diff
@@ -9,15 +9,23 @@
 <div id="remote-realm-confirm-login-page" class="register-account flex full-page">
     <div class="center-block new-style">
         <div class="pitch">
-            <h1>Log in to Zulip server billing for {{ remote_realm_host }}</h1>
+            <h1>Log in to Zulip server billing for {{ host }}</h1>
         </div>
         <div class="white-box">
             <p>Click <b>Continue</b> to log in to Zulip server
             billing with the account below.</p>
+            {% if not legacy_server_confirmation_flow %}
             Full name: {{ user_full_name }}<br />
+            {% endif %}
```

Comment:
I think we can just drop the full name here unconditionally once you've done the `RemoteRealm` variant of this.